pkg/pillar/scripts/device-steps.sh

```diff
@@ -330,6 +333,9 @@ if ! pgrep ledmanager >/dev/null; then
     echo "$(date -Ins -u) Starting ledmanager"
     ledmanager &
 fi
+sleep 10
```

Comment:
No need for random sleep. If you had to insert this, then your code has some race conditions which need to be fixed.

Reply:
removed